app/build.gradle

```diff
@@ -1,43 +1,54 @@
 apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion rootProject.ext.compileSdkVersion
 
```

Comment:
Could move `namespace` here too and remove from the manifest, to get it ready for AGP 8 https://developer.android.com/studio/build/configure-app-module#set-namespace

Reply:
done!